docker-compose.yml

```diff
@@ -12,15 +12,16 @@ services:
     - db
     - mailcatcher
     - cache
-    - webpack
```

Comment:
Da stimme ich nicht ganz zu. Die Dependencies kommen ja nicht zum Zug wenn man `docker-compose up` macht. Nur wenn man `docker-compose up rails` macht, entscheidet `depends_on`, welche anderen Services auch gestartet werden. Wenn du in diesem Szenario `webpack` nicht willst, brauchen wir wohl auch `mailcatcher` nicht.